test/staged.jl

```diff
@@ -174,7 +174,7 @@ let gf_err, tsk = @async nothing # create a Task for yield to try to run
     end
     @test_throws ErrorException gf_err()
     @test_throws ErrorException gf_err()
-    @test gf_err_ref[] == 3
+    @test gf_err_ref[] == 4
```

Comment:
The value is unimportant, it just tells how many times we attempted to and failed to run the generated function and is useful for being aware of when it changes